proc_IR.py

```diff
@@ -11,27 +11,35 @@
 clock_correction = 1.785
 filter_bandwidth = 5e3
 coh_sel = {'ph1':0,
-        'ph2':1}
+        'ph2':0}
 coh_err = {'ph1':1,# coherence channels to use for error
         'ph2':r_[0,2,3]}
 # }}}
 
 for searchstr,exp_type,nodename, postproc in [
-        ('200212_IR_3_30dBm', 'test_equip', 'signal', 
-            'spincore_IR_v1'),
+        #('w8_200731', 'test_equip', 2, 
+        #    'ag_IR2H'),
+        ('w8_200309', 'test_equip',2,'ag_IR2H'),
         ]:
     fl.basename = searchstr
-    s = find_file(searchstr, exp_type=exp_type,
+    if postproc=='ag_IR2H':
+        s = find_file(searchstr,exp_type=exp_type,
+                expno=nodename,
+                postproc=postproc, lookup=postproc_dict,
+                dimname='indirect')
+    if postproc=='spincore_IR_v1':    
+        s = find_file(searchstr, exp_type=exp_type,
             expno=nodename,
             postproc=postproc, lookup=postproc_dict,
-            dimname='indirect')
-    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
+            clock_correction=clock_correction, dimname='indirect')
     #{{{filter data
-    s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
+    if postproc=='spincore_IR_v1':
+        s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
```

Comment:
I also commented on this in proc 90, so maybe go look there?

Reply:
So this guy has the if statements based on if they use the clock correction. If it is just ag_IR2h there is no need for an input of clock correction for the preprocessing however, if it is spincore_IR then the clock correction needs to be one of the kwarg for preprocessing.

Reply:
I'm confused what that has to do with the current line -- this is about the filter bandwidth, not the clock correction

Reply:
fixed